README.md

```diff
@@ -50,6 +50,18 @@ dep you should use the [govendor tool][govendor].
 Development currently targets Go version 1.6 but will probably work on 1.5 with
 the vendor experiment enabled.
 
+### Android
+To work on the Android app do the following:
+
+* Install [gomobile][gomobile].
+* Install [Android Studio][android] and SDK level 22.
+  * Connect a device and ensure you can see it with `adb devices`.
+* In Android Studio set your `GOPATH` by going to `Preferences > Build,
```

Comment:
In my Android Studio (on Arch Linux), it's `File > Settings` instead of `Preferences`